index.html

```diff
@@ -1,52 +1,3336 @@
+              </div>
+              <div class="outer_q102">
+              <div
+                id="q102"
+                data-name="Group 23"
+                class="q102"
+              >
+                <div class="outer_q103">
+                <div
+                  id="q103"
+                  data-name="Rectangle 27"
+                  
+                  class="q103"
+                >
+                </div>
+                </div>
+                <div class="outer_q104">
+                <div
+                  id="q104"
+                  data-name="Ruby on Rails"
+                  class="q104"
+                ><div>Ruby on Rails</div>
+                </div>
+                </div>
+              </div>
+              </div>
+              <div class="outer_q105">
+              <div
+                id="q105"
+                data-name="Rectangle 27"
+                
+                class="q105"
+              >
+              </div>
+              </div>
+              <div class="outer_q106">
+              <div
+                id="q106"
+                data-name="HTML/CSS"
+                class="q106"
+              ><div>HTML/CSS</div>
+              </div>
+              </div>
+              <div class="outer_q107">
+              <div
+                id="q107"
+                data-name="Group 19"
+                class="q107"
+              >
+                <div class="outer_q108">
+                <div
+                  id="q108"
+                  data-name="Rectangle 20"
+                  class="q108"
+                >
+                </div>
+                </div>
+                <div class="outer_q109">
+                <img
+                  id="q109"
+                  data-name="ic_arrow_right"
+                  src="images/ic_arrow_right.png"
+                  alt="ic_arrow_right"
+                  class="q109"
+                >
+                </div>
+                <div class="outer_q110">
+                <div
+                  id="q110"
+                  data-name="See this project"
+                  class="q110"
+                ><div>See this project</div>
+                </div>
+                </div>
+              </div>
+              </div>
+            </div>
+            </div>
+          </div>
+          </div>
+          <div class="outer_q111">
+          <div
+            id="q111"
+            data-name="HTML/CSS"
+            class="q111"
+          ><div>HTML/CSS</div>
+          </div>
+          </div>
+        </div>
+        </div>
+        <div class="outer_q112">
+        <div
+          id="q112"
+          data-name="Group 118"
+          class="q112"
+        >
+          <div class="outer_q113">
+          <div
+            id="q113"
+            data-name="Group 49"
+            class="q113"
+          >
+            <div class="outer_q114">
+            <div
+              id="q114"
+              data-name="About"
+              class="q114"
```

Comment:
- [ ] Kindly, you are repeating your code multiple times, please do check the unnecessary code to be removed. 
- [ ] The container of the section is a section tag, please  make sure that the container is a section to keep semantic code in our project.
- [ ]  The main container of the projects has a display:grid, please use display grid to display our projects work as it required in the project requirements
- [ ]  The button (e.g., "See This project") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design), kindly make see this project as a link or button with a hover color.
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li, kindly use ul and li to list the technologies.

Reply:
Thanks for your review and kindly information. I took the advantage to rush my code for the next week because I already repeated the first one :-(. All appointments you required me I'll do and prepare for the next round of exercise. 
I duplicate too many lines because I was (I'm) be wared to lose this opportunity. But I'll fix all of them.

Just a question: To get your approval I just need to remove the project requirements in my Read me file?